terraform/development/dynamodb.tf

```diff
@@ -1,24 +1,24 @@
 resource "aws_dynamodb_table" "personapi_dynamodb_table" {
-    name                  = "Persons"
-    billing_mode          = "PROVISIONED"
-    read_capacity         = 10
-    write_capacity        = 10
-    hash_key              = "id"
+  name           = "Persons"
+  billing_mode   = "PROVISIONED"
+  read_capacity  = 10
+  write_capacity = 10
+  hash_key       = "id"
 
-    attribute {
-        name              = "id"
-        type              = "S"
-    }
+  attribute {
+    name = "id"
+    type = "S"
+  }
 
-    tags = {
-        Name              = "person-api-${var.environment_name}"
-        Environment       = var.environment_name
-        terraform-managed = true
-        project_name      = var.project_name
-        BackupPolicy     = "Dev"
-    }
+  tags = {
+    Name              = "person-api-${var.environment_name}"
```

Comment:
I think you should add the default tags to the main.tf (like you did for the tenure-api) to keep it consistent